src/services/ghost/strategies/ErrorFixStrategy.ts

```diff
@@ -1,9 +1,25 @@
-import * as vscode from "vscode"
+import type { Diagnostic } from "vscode"
 import { GhostSuggestionContext } from "../types"
 import { UseCaseType } from "../types/PromptStrategy"
 import { BasePromptStrategy } from "./BasePromptStrategy"
 import { CURSOR_MARKER } from "../ghostConstants"
 
+// Local constants for DiagnosticSeverity values since we're using type-only imports
+const DiagnosticSeverityValues = {
+	Error: 0,
+	Warning: 1,
+	Information: 2,
+	Hint: 3,
+} as const
+
+// Mapping for getting severity names from values
+const DiagnosticSeverityNames: Record<number, string> = {
+	0: "Error",
+	1: "Warning",
+	2: "Information",
+	3: "Hint",
+}
```

Comment:
Can we move these into a new file where they can be referenced from multiple files? 
`./diagnostics.ts` or something?

Reply:
done!